client/me/help/help-contact/index.jsx

```diff
@@ -582,16 +598,27 @@ const HelpContact = React.createClass( {
 		return SUPPORT_HAPPYCHAT !== variationSlug && SUPPORT_LIVECHAT !== variationSlug && null != ticketSupportRequestError;
 	},
 
-	shouldShowPreloadForm: function() {
-		const { olark, sitesInitialized } = this.state;
+	/**
+	 * Before determining which variation to assign, certain async data needs to be in place.
+	 * This function helps assess whether we're ready to say which variation the user should see.
+	 *
+	 * @returns {Boolean} Whether all the data is present to determine the variation to show
+	 */
+	hasDataToDetermineVariation: function() {
```

Comment:
Would it make sense to negate this function? I'm wondering if something like `isDeterminingVariation` would be easier to understand in the boolean logic where it's later used, eg:

```
return ! sitesInitialized || this.isDeterminingVariation() || waitingOnDirectly;
```